scripts/userstyles.yml

```
@@ -336,6 +339,14 @@ userstyles:
       app-link: "https://mail.google.com"
     current-maintainers: []
     past-maintainers: [*isabelroses]
+  go-dev:
+    name: Go.dev
```

```suggestion
    name: go.dev
```
Either `go.dev` or `The Go Programming Language` works I think.

---

Would like to stick with `go.dev`. I'm just concerned that there might be some confusion with the `catppuccin/go` in the website.